packages/runner/src/jsonrpc_server.ts

```diff
@@ -0,0 +1,88 @@
+import { ChainService } from "@ckb-godwoken/godwoken";
+import jayson from "jayson/promise";
+import cors from "cors";
+import connect from "connect";
+import { json } from "body-parser";
+
+function isHexString(s: any) {
+  return typeof s === "string" && /^0x([0-9a-fA-F][0-9a-fA-F])*$/.test(s);
+}
+
+function isHash(s: any) {
+  return isHexString(s) && s.length === 66;
+}
+
+export class JsonrpcServer {
+  chainService: ChainService;
+  server: jayson.Server;
+  listen: string;
+
+  constructor(chainService: ChainService, listen: string) {
+    this.chainService = chainService;
+    this.server = new jayson.Server({
+      ckb_submitL2Transaction: this.submitL2Transaction.bind(this),
+      ckb_executeL2Tranaction: this.executeL2Transaction.bind(this),
+      ckb_submitWithdrawalRequest: this.submitWithdrawalRequest.bind(this),
+      ckb_getBalance: this.getBalance.bind(this),
+      ckb_getStorageAt: this.getStorageAt.bind(this),
+      ckb_getAccountIdByScriptHash: this.getAccountIdByScriptHash.bind(this),
```

Comment:
Why the prefix are all `ckb_` ?

Reply:
It's to make a clear distinction between them and Ethereum RPCs: https://eth.wiki/json-rpc/API

At later time we might implement Ethereum RPCs in polyjuice for better developer experience. Having prefixes here allows us to tell ETH ones from CKB ones

Reply:
I think it's better use `gw_` prefix, since those are all godwoken related RPC.

Reply:
Yeah that makes sense, will change them